screencaptor/src/main/java/com/wealthfront/screencaptor/ScreenCaptor.kt

```diff
@@ -69,8 +70,8 @@ object ScreenCaptor {
    */
   @Synchronized
   fun takeScreenshot(
-    view: View,
-    screenshotFilename: String,
+    rootViews: List<View>,
+    screenshotName: String,
```

Comment:
minor thing: I modified screenshotFilename -> screenshotName. Since we do some logic inside the method to derive the file name, IMO this is more like an ID/nickname for the screenshot.

Reply:
Can you update the readme as well? :)